_posts/2022/2022-01-26-dev-experience-2022.md

```diff
@@ -0,0 +1,85 @@
+
+## The Second Semester
+
+#### The initiation
+
+I got a phone call informing me about my induction to SDSLabs. We had an intro chapo (online, coz covid) and got to know everyone there. We were added on the workspace apps and this got the cart rolling.
+
+The semester was filled with lectures on various subjects (Git, Bash scripting, Web development, Game development) and corresponding assignments. The assignments spurred me to learn about something new and implement it within a deadline.
+
+After the assignment season was concluded, we were introduced to the active apps and projects in SDSLabs. We were briefed on the status for each project and a received an overview of the tech stack, ongoing work, future issues and overall vision.
+
+#### Minors!!
+
+The day came when we were asked to pick up a project from the ongoing ones to complete a minor issue in them. This was sorting-hat-esque where everyone looked at that spreadsheet of projects and open issues and put their name against any one of them. Minors were completed in less than a week.
+
+## The Third Semester
+
+#### Hackathons
+
+By now we started participating in various hackathons (*and managing to win some of them!!*). Members would come together, form teams among themselves (sometimes even our alumni joined us) and we would participate in the hackathons. Sitting through hours of code sprints and preparing POCs, those were exhilarating hours.
+
+#### Majors
+After the minors, we had to choose our Majors: solving a large issue or implementing something important in the ongoing projects. We again met for the sorting and were assigned the responsibilities.
+
+#### Releasing a Project
+
+When a project comes to fruition and is ready to be open-sourced, everyone at SDSLabs is pretty excited about it. Seeing the hardwork of our members getting released ittruly a moment to be cherished. I witnessed the release of [ZapDB](https://github.com/sdslabs/zap-db) and [Studyportal](https://github.com/sdslabs/studyportal).
+
+#### Not just Dev
+
+It's not always software develpoment that we work on. We also work on IoT projects to make our workspace smarter (like presence detection using computer vision, WiFi locks). We have members passionate about information security and Data Science. We also have paper discussions in which we all read a research paper on related to CS and discuss it with other members.
```

Comment:
iske end me add kar de apna CSAW experience as an example

Reply:
Ig you can even write fun things like playing Minecraft together? :p

Reply:
minesweeper aur minecraft aur nioh ka soch rha tha lekin lamba ho jaata